convert2rhel/utils.py

```diff
@@ -702,44 +704,51 @@ def is_dir_empty(path):
             os.rmdir(path)
 
 
-def hide_secrets(args, secret_args=frozenset(("--password", "--activationkey", "--token", "-p", "-k"))):
+def hide_secrets(
+    args, secret_options=frozenset(("--username", "--password", "--activationkey", "--org", "-u", "-p", "-k", "-o"))
+):
     """
-    Replace secret values with asterisks.
+    Replace secret values passed through a command line with asterisks.
 
-    This function takes a list of arguments which will be passed
-    in a transformation process where we will replace any secret values
-    with an fixed size of asterisks (*) and returns a new list containing
-     the arguments with this transformation.
+    This function takes a list of command line arguments and returns a new list containing where any secret value is
+    replaced with a fixed size of asterisks (*).
 
-    :arg args: An argument list which may contain secret values.
+    Terminology:
+     - an argument is one of whitespace delimited strings of an executed cli command
+       Example: `ls -a -w 10 dir` ... four arguments (-a, -w, 10, dir)
+     - an option is a subset of arguments that start with - or -- and modify the behavior of a cli command
+       Example: `ls -a -w 10 dir` ... two options (-a, -w)
+     - an option parameter is the value following an aption if the option requires a value
+       Example: `ls -a -w 10 dir` ... one option parameter (10)
+
+    :arg args: A list of command line arguments which may contain secret values.
+    :arg secret_options: A set of command line options requiring sensitive or private information.
```

Comment:
I would write those as `:param:` instead of `:arg:` as it seems more appropriate with the rest of the codebase (at least, the ones who have docstrings that documents the parameters). 

Although, not wrong if they stay in :arg: as we are not using any type of document generation. Up to you to, then.